es-vue-base/src/lib-icons/icon-lightbulb.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <svg
+        :style="{
+            height: height,
+            width: width,
+        }"
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 24 24"
+        fill="currentColor">
+        <path d="M19.6413 8.0574c-.2908-3.57905-3.0869-6.52059-6.6324-6.99034-2.2369-.301982-4.47385.38028-6.14034 1.84545C5.19087 4.37769 4.229 6.49157 4.229 8.71729c0 2.71781 1.45399 5.26791 3.79156 6.65481.50331.302.50331.4138.50331.5816v1.0625c0 .4586.38027.8389.83884.8389.45857 0 .83889-.3803.83889-.8389v-1.0625c0-1.0961-.67112-1.633-1.33101-2.0244-1.83427-1.0849-2.97509-3.0758-2.97509-5.21201 0-1.74479.74937-3.4001 2.05796-4.55211 1.33096-1.16319 3.04224-1.6665 4.80934-1.44281 2.729.35791 4.9548 2.70666 5.1896 5.45806.2014 2.42707-1.0289 4.63037-3.1987 5.74887-.4251.2237-1.4093.727-1.4428 2.0356v1.029c0 .4585.3802.8388.8388.8388s.8388-.3803.8388-.8388v-1.0067c0-.1677 0-.2908.5369-.5704 2.7961-1.4428 4.362-4.2725 4.1047-7.38177l.0112.02237ZM14.4516 18.7499H9.45212c-.45856 0-.83884.3803-.83884.8388 0 .4586.38028.8389.83884.8389h4.99948c.4586 0 .8389-.3803.8389-.8389 0-.4585-.3803-.8388-.8389-.8388ZM14.4516 21.3223H9.45212c-.45856 0-.83884.3802-.83884.8388s.38028.8388.83884.8388h4.99948c.4586 0 .8389-.3802.8389-.8388s-.3803-.8388-.8389-.8388Z" />
+    </svg>
+</template>
+<script>
+export default {
+    name: 'IconLightbulb',
```

Comment:
nit: isn't light bulb two words? Should this be `icon-light-bulb.vue` with the name `IconLightBulb`?

Reply:
Went down a bit of a rabbit hole.
The modern usage is "light bulb" (with space) but some dictionaries took a while to get with the times so while "lightbulb" (no space) is generally acceptable it is not the preferred.

Reply:
In the spirit of modernization, the names are now `icon-light-bulb.vue` and `IconLightBulb` 💡

Reply:
This `icon-lightbulb.vue` should probably be deleted now that we have the new one.